test/ArcxAnalyticsSdk.test.ts

```diff
@@ -38,13 +39,17 @@ const ALL_FALSE_CONFIG: Omit<SdkConfig, 'url'> = {
   trackWalletConnections: false,
   trackChainChanges: false,
   trackTransactions: false,
+  trackSigning: false,
 }
 
 describe('(unit) ArcxAnalyticsSdk', () => {
   let postRequestStub: sinon.SinonStub
 
   beforeEach(() => {
     window.ethereum = sinon.createStubInstance(MetaMaskInpageProvider)
+    window.web3 = {
+      currentProvider: window.ethereum,
+    }
```

Comment:
is this really needed? `getWeb3Provider()` has implementation of:
```
function getWeb3Provider(): MetaMaskInpageProvider | undefined {
  return window.web3?.currentProvider || window?.ethereum
}
```

Reply:
And inside tests, you can keep using `window.ethereum`

Reply:
done